mne/decoding/time_gen.py

```diff
@@ -256,7 +267,7 @@ def f(x):
         self.estimators_ = sum(out, list())
         return self
 
-    def predict(self, epochs, test_times=None, picks=None):
+    def predict(self, epochs, test_times=None, y=None, picks=None):
```

Comment:
passing a y to a predict method is impossible to explain to a user...
especially with some sklearn background. Please don't be creative...